nss-fumbling-penguins/nutshell
# added favicon

## Link to Ticket
Closes # [Link]()

## Description of Proposed Changes

adds a favicon to the tab of the page

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout rm-add-favicon

```
1. pull down branch
1. check for new png file in src directory
1. check for new copy option command in gruntfile.js to copy the favicon to dist
1. close and re run grunt
1. make sure favicon was copied to your dist directory
1. reload the page and make sure favicon is displayed in the tab

## Impacted Areas in Application

List general components of the application that this PR will affect:

index html file

## Mentions @username

Tag users that need to review this code

## Definition of Done

1. The project must be fully documented. This includes the following:
    1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
    1. Every class must be documented with purpose, author, and methods.
    1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
